CHANGES.rst

```diff
@@ -7,6 +7,7 @@ unreleased
 
 - allow to set the timeout and retry interval when waiting for services
 - added explicit dependency to six, which is no more required in newer pytest versions
+- add support for `docker-compose stop`
```

Comment:
can you pls always add new entries on top, so lower lines are never modified, thx